docs/pages/get-started/faq.md

```diff
@@ -384,21 +405,13 @@ Clients communicate with XMTP nodes through a gRPC (or JSON/HTTP) message API.
 
 XMTP provides perceptibly real-time message delivery and retrieval. The network does not provide service level guarantees.
 
-### Is there a way to get a list of all current XMTP-enabled wallets?
-
-XMTP doesn't provide a direct method to fetch all XMTP-enabled wallet addresses in bulk. You might consider using a third-party service to do so instead of building the functionality yourself.
-
-For example, services like [Blaze](https://www.withblaze.app/) have compiled extensive lists of XMTP addresses by querying the XMTP `canMessage` API over time at a rate that avoids hitting the limits.
-
-### Does XMTP support group chat?
+## Fees
 
-Yes. XMTP V3 supports group chat in React Native, Android, iOS, and Node.js SDKs.
+XMTP core developers and researchers are working on outlining a specific fee model for XMTP. The team will share a proposed plan as soon as it’s well-defined enough to generate meaningful input.
```

Comment:
I like the idea in the other doc about this being more an invitation to discussion than a promise of something later.

Reply:
Thank you for this feedback, @saulmc - sounds great! I will make the forum topic public and update this section to just point to it.

Reply:
@neekolas has what seems like a workable approach mapped out, I wonder if we could put two sentences here + invitation to discuss. Example:

XMTP core developers and researchers are working on a specific fee model for XMTP, with the following guiding principles in mind:
- Infrastructure costs for the network must remain low even when decentralized, and comparable to the costs for an equivalent centralized messaging service.
- There must be a sizable free tier for most end users, with the low cost per-user subsidized by the largest apps as well as bulk senders.
- There must be a low 'take rate': the biggest driver of cost must be infrastructure costs.

Reply:
Thank you for this feedback as well, @saulmc and @richardhuaaa! I feel like I hit the doc review jackpot today! I've updated the content as follows: https://github.com/xmtp/docs-xmtp-org/pull/26/files#diff-0f5558e2ccc886a61098eac3e4020166d6c4dee87249fc7dcda8eba0f5bd1196R414

Reply:
I like that suggestion, but I wouldn't set unreasonable expectations around a "free tier". That's really up to the applications and ecosystem to determine. The protocol expects $$$.

Maybe we just strike that part and replace with: "Fees will be dynamic to protect the network from Denial Of Service attacks"

Reply:
Agreed. Free tier is not part of the protocol design.